node/node.go

```diff
@@ -1197,7 +1204,7 @@ func (app *App) listenToUpdates(ctx context.Context) {
 						if err := atxsync.Download(
 							ctx,
 							10*time.Second,
-							app.addLogger(SyncLogger, app.log).Zap(),
+							app.syncLogger.Zap(),
```

Comment:
Instead of remembering the `syncLogger` in the `App`, you could create it out of the goroutine, at the very top of `listenToUpdates()`. It would create a second logger with the same name but I don't think it's a problem, is it?

Reply:
It does create a new instance of the logger with the same name. I don't think it is a problem in terms of name collisions or similar, but it _might_ be in terms of performance. Every other logger we use we also only instantiate once and some of those we pass to multiple users (e.g. `NipostValidatorLogger` or the `Fetcher` logger) and I wanted to keep in line with this.

Alternatively we could change `addLogger` to return the same instance if it is called multiple times with the same name? We will have to revisit this when we enable multi-smeshers anyway to make sure the right logs are printed with the correct nodeId (and logs from GRPC and DB are not - since there will only be one instance of each of these services).